components/automate-ui/src/app/components/page-picker/page-picker.component.html

```diff
@@ -1,35 +1,83 @@
 <ng-container *ngIf="selectablePages.length > 1">
-  <chef-button secondary
-    class="page-picker-item page-picker-first"
-    (click)="onItemTap(first)"
-    [disabled]="isNull(prev)">
-    <chef-icon>first_page</chef-icon>
-  </chef-button>
-  <chef-button secondary
-    class="page-picker-item page-picker-prev"
-    (click)="onItemTap(prev)"
-    [disabled]="isNull(prev)">
-    <chef-icon>chevron_left</chef-icon>
-  </chef-button>
-  <chef-button
-    *ngFor="let item of selectablePages; trackBy: trackBy"
-    class="page-picker-item"
-    [attr.primary]="item === page"
-    [attr.secondary]="item !== page"
-    [attr.disabled]="selectablePages.length <= 1"
-    (click)="onItemTap(item)">
-    {{ item }}
-  </chef-button>
-  <chef-button secondary
-    class="page-picker-item page-picker-next"
-    (click)="onItemTap(next)"
-    [disabled]="isNull(next)">
-    <chef-icon>chevron_right</chef-icon>
-  </chef-button>
-  <chef-button secondary
-    class="page-picker-item page-picker-last"
-    (click)="onItemTap(last)"
-    [disabled]="isNull(next)">
-    <chef-icon>last_page</chef-icon>
-  </chef-button>
+  <ng-container *ngIf="!forDesktop">
+    <chef-button secondary
+      class="page-picker-item page-picker-first"
+      (click)="onItemTap(first)"
+      [disabled]="isNull(prev)">
+      <chef-icon>first_page</chef-icon>
+    </chef-button>
+    <chef-button secondary
+      class="page-picker-item page-picker-prev"
+      (click)="onItemTap(prev)"
+      [disabled]="isNull(prev)">
+      <chef-icon>chevron_left</chef-icon>
+    </chef-button>
+    <chef-button
+      *ngFor="let item of selectablePages; trackBy: trackBy"
+      class="page-picker-item"
+      [attr.primary]="item === page"
+      [attr.secondary]="item !== page"
+      [attr.disabled]="selectablePages.length <= 1"
+      (click)="onItemTap(item)">
+      {{ item }}
+    </chef-button>
+    <chef-button secondary
+      class="page-picker-item page-picker-next"
+      (click)="onItemTap(next)"
+      [disabled]="isNull(next)">
+      <chef-icon>chevron_right</chef-icon>
+    </chef-button>
+    <chef-button secondary
+      class="page-picker-item page-picker-last"
+      (click)="onItemTap(last)"
+      [disabled]="isNull(next)">
+      <chef-icon>last_page</chef-icon>
+    </chef-button>
+  </ng-container>
+
+  <!-- new styles being used only for desktop at the moment -->
+  <ng-container *ngIf="forDesktop">
```

Comment:
Initially this was just for desktop and contained the new styling, so I added an attribute that provided an entirely different look.  We've since held off on styling, so I removed the new styles but am not yet sure if I should remove the current layout of pagination.  This is allowing us to do both until further direction.